Error: Argument actual was shadowed by a local variable before it was used.

A couple of PR (https://github.com/wort-schule/wort.schule/pulls?q=is%3Apr+is%3Aopen+label%3Adependencies) fail to run tests. The problem:

```
spec/support/open_graph_matcher.rb:13:5: W: Lint/ShadowedArgument: Argument actual was shadowed by a local variable before it was used.
[10](https://github.com/wort-schule/wort.schule/actions/runs/3445139675/jobs/5748510886#step:5:11)
    actual = first("meta[property='#{name}']", visible: false)
[11](https://github.com/wort-schule/wort.schule/actions/runs/3445139675/jobs/5748510886#step:5:12)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[12](https://github.com/wort-schule/wort.schule/actions/runs/3445139675/jobs/5748510886#step:5:13)

[13](https://github.com/wort-schule/wort.schule/actions/runs/3445139675/jobs/5748510886#step:5:14)
171 files inspected, 1 offense detected
[14](https://github.com/wort-schule/wort.schule/actions/runs/3445139675/jobs/5748510886#step:5:15)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/run_tests.yml
+++ b/.github/workflows/run_tests.yml
@@ -29,7 +29,7 @@ jobs:
           bundler-cache: true
 
       - name: Run Ruby lint
-        run: bundle exec standardrb
+        run: bundle exec standardrb --no-fix
 
       - name: Install libvips
         env:

--- a/app/decorators/page_decorator.rb
+++ b/app/decorators/page_decorator.rb
@@ -9,5 +9,4 @@ class PageDecorator < ApplicationDecorator
   #       object.created_at.strftime("%a %m/%d/%y")
   #     end
   #   end
-
 end

--- a/config/deploy.rb
+++ b/config/deploy.rb
@@ -1,6 +1,6 @@
-require 'mina/rails'
-require 'mina/git'
-require 'mina/rvm'
+require "mina/rails"
+require "mina/git"
+require "mina/rvm"
 
 # Basic settings:
 #   domain       - The hostname to SSH to.
@@ -8,17 +8,17 @@ require 'mina/rvm'
 #   repository   - Git repo to clone from. (needed by mina/git)
 #   branch       - Branch name to deploy. (needed by mina/git)
 
-set :application_name, 'wortschule'
-set :domain, 'wort.schule'
+set :application_name, "wortschule"
+set :domain, "wort.schule"
 set :user, fetch(:application_name)
 set :deploy_to, "/home/wortschule/app"
-set :repository, 'git@github.com:wort-schule/wort.schule.git'
-set :branch, 'main'
-set :rvm_use_path, '/etc/profile.d/rvm.sh'
+set :repository, "git@github.com:wort-schule/wort.schule.git"
+set :branch, "main"
+set :rvm_use_path, "/etc/profile.d/rvm.sh"
 
 # Optional settings:
 #   set :user, 'foobar'          # Username in the server to SSH to.
-set :user, 'wortschule'
+set :user, "wortschule"
 #   set :port, '30000'           # SSH port number.
 #   set :forward_agent, true     # SSH forward_agent.
 
@@ -26,44 +26,41 @@ set :user, 'wortschule'
 # Some plugins already add folders to shared_dirs like `mina/rails` add `public/assets`, `vendor/bundle` and many more
 # run `mina -d` to see all folders and files already included in `shared_dirs` and `shared_files`
 # set :shared_dirs, fetch(:shared_dirs, []).push('public/assets')
-set :shared_files, fetch(:shared_files, []).push('config/database.yml', 'config/secrets.yml')
-set :shared_dirs, fetch(:shared_dirs, []).push('public/packs', 'node_modules', 'storage')
+set :shared_files, fetch(:shared_files, []).push("config/database.yml", "config/secrets.yml")
+set :shared_dirs, fetch(:shared_dirs, []).push("public/packs", "node_modules", "storage")
 
 # This task is the environment that is loaded for all remote run commands, such as
 # `mina deploy` or `mina rake`.
 task :remote_environment do
-  ruby_version = File.read('.ruby-version').strip
+  ruby_version = File.read(".ruby-version").strip
   raise "Couldn't determine Ruby version: Do you have a file .ruby-version in your project root?" if ruby_version.empty?
 
-  invoke :'rvm:use', ruby_version
+  invoke :"rvm:use", ruby_version
 end
 
 # Put any custom commands you need to run at setup
 # All paths in `shared_dirs` and `shared_paths` will be created on their own.
 task :setup do
-
   in_path(fetch(:shared_path)) do
-
-    command %[mkdir -p config]
+    command %(mkdir -p config)
 
     # Create database.yml for Postgres if it doesn't exist
     path_database_yml = "config/database.yml"
-    database_yml = %[production:
+    database_yml = %(production:
   database: #{fetch(:user)}
   adapter: postgresql
   pool: 5
-  timeout: 5000]
-    command %[test -e #{path_database_yml} || echo "#{database_yml}" > #{path_database_yml}]
+  timeout: 5000)
+    command %(test -e #{path_database_yml} || echo "#{database_yml}" > #{path_database_yml})
 
     # Create secrets.yml if it doesn't exist
     path_secrets_yml = "config/secrets.yml"
-    secrets_yml = %[production:\n  secret_key_base:\n    #{`bundle exec rake secret`.strip}]
-    command %[test -e #{path_secrets_yml} || echo "#{secrets_yml}" > #{path_secrets_yml}]
-    
-    # Remove others-permission for config directory
-    command %[chmod -R o-rwx config]
-  end
+    secrets_yml = %(production:\n  secret_key_base:\n    #{`bundle exec rake secret`.strip})
+    command %(test -e #{path_secrets_yml} || echo "#{secrets_yml}" > #{path_secrets_yml})
 
+    # Remove others-permission for config directory
+    command %(chmod -R o-rwx config)
+  end
 end
 
 desc "Deploys the current version to the server."
@@ -73,12 +70,12 @@ task :deploy do
   deploy do
     # Put things that will set up an empty directory into a fully set-up
     # instance of your project.
-    invoke :'git:clone'
-    invoke :'deploy:link_shared_paths'
-    invoke :'bundle:install'
-    invoke :'rails:db_migrate'
-    invoke :'rails:assets_precompile'
-    invoke :'deploy:cleanup'
+    invoke :"git:clone"
+    invoke :"deploy:link_shared_paths"
+    invoke :"bundle:install"
+    invoke :"rails:db_migrate"
+    invoke :"rails:assets_precompile"
+    invoke :"deploy:cleanup"
 
     on :launch do
       command "sudo systemctl restart #{fetch(:user)}"

--- a/config/initializers/friendly_id.rb
+++ b/config/initializers/friendly_id.rb
@@ -16,9 +16,9 @@ FriendlyId.defaults do |config|
   # undesirable to allow as slugs. Edit this list as needed for your app.
   config.use :reserved
 
-  config.reserved_words = %w(new edit index session login logout users admin
-    stylesheets assets javascripts images)
-    
+  config.reserved_words = %w[new edit index session login logout users admin
+    stylesheets assets javascripts images]
+
   # This adds an option to treat reserved words as conflicts rather than exceptions.
   # When there is no good candidate, a UUID will be appended, matching the existing
   # conflict behavior.

--- a/spec/decorators/page_decorator_spec.rb
+++ b/spec/decorators/page_decorator_spec.rb
@@ -1,4 +1,4 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe PageDecorator do
 end

--- a/spec/requests/page_spec.rb
+++ b/spec/requests/page_spec.rb
@@ -1,4 +1,4 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe "Pages", type: :request do
   describe "GET /impressum" do
@@ -7,5 +7,4 @@ RSpec.describe "Pages", type: :request do
       expect(response).to have_http_status(:success)
     end
   end
-
 end

--- a/spec/support/open_graph_matcher.rb
+++ b/spec/support/open_graph_matcher.rb
@@ -10,10 +10,10 @@ RSpec::Matchers.define :have_open_graph do |name, expected|
   end
 
   failure_message do |actual|
-    actual = first("meta[property='#{name}']", visible: false)
+    actual_element = first("meta[property='#{name}']", visible: false)
 
-    if actual
-      "expected that meta #{name} would have content='#{expected}' but was '#{actual[:content]}'"
+    if actual_element
+      "expected that meta #{name} would have content='#{expected}' but was '#{actual_element[:content]}'"
     else
       "expected that meta #{name} would exist with content='#{expected}'"
     end

